internal/internal/src/footer.tsx

```diff
@@ -7,43 +7,62 @@ import {
   ServiceProviderBadge,
   TelegramIcon,
   TwitterIcon,
-} from "../1 - atoms";
+} from "./icons";
 
 const footerProducts = [
   {
-    name: "NameKit",
-    href: "https://namekit.io",
+    name: "ENSNode",
+    href: "https://www.ensnode.io/",
+  },
+  {
+    name: "ENS Referral Program",
+    href: "https://namehashlabs.org/ens-referral-program",
+  },
+  {
+    name: "NameAI",
+    href: "https://nameai.dev/",
```

Comment:
```suggestion
    href: "https://nameai.io/",
```

We have this at a new domain now